src/app.js

```diff
@@ -64,9 +70,9 @@ if (__DEV__) {
   // $FlowFixMe
   console.info = log
   // $FlowFixMe
-  console.warn = log
+  // console.warn = log
```

Comment:
The `log` method is/was supposed to write the logs to disk, so dropping this will break that feature. However, I see from the git blame that Paul changed `if (!__DEV__)` condition guarding this block to `if (__DEV__)` in his "Add activity log" PR. This seems like a mistake. So, unless there is some context we are missing here, I think the correct fix is to put `if (!__DEV__)` back as it was, to leave the logs unmodified while debugging the app.

Otherwise, if the `log` feature truly has been replaced by the activity logs, we should just delete this whole block of code, and the `log` function itself as well, since we don't want it either in production or debugging.

Reply:
I talked to @samholmes  about this and yes the line should be put back to !__DEV__

It was changed because I needed to debug that condition but it snuck into the PR. What Sam was going to do was understand how the production build was still properly getting logs at all. This change would imply only dev builds would get the uploaded logs

Reply:
Though what's confusing is how this "works" in production. Is this block needed at all?